src/lib/module.html

```diff
@@ -44,15 +44,21 @@
       dependencies = Array.isArray(id) ? id : [];
     }
 
+    var inferredId = _inferModuleId();
     if (typeof id !== 'string') {
-      id = _inferModuleId();
+      id = inferredId;
+    }
+    if (id.indexOf('\\') !== -1) {

```

Comment:
I'd like to actually support it in the name of browser compatibility (AFAIK everyone but Firefox rewrites `\` into `/` anyway). I'll stick a TODO on there